### PR TITLE
Kong Stop

### PR DESCRIPTION
**Pull request by thefosk**
This branch is a test to try to fix the `STOP` problem that afflicts Kong.

Sometimes nginx exits, but the ports are still open. Trying to figure out why.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -17,6 +17,7 @@ before_install:
 install:
   - sudo make install
   - sudo make dev
+  - sudo sed -i.bak s@/usr/local/bin/luajit@/usr/bin/lua@g /usr/local/bin/busted
```
Comment (thefosk) [inline .travis.yml]: Temporary fix for busted segmentation fault bug.

File: spec/unit/stub_coverage_spec.lua [hunk 2]
```diff
@@ -6,7 +6,7 @@ local IO = require "kong.tools.io"
 -- Stub DAO for lapis controllers
 _G.dao = {}
 
-local lua_sources = IO.retrieve_files("./kong", { exclude_dir_patterns = {"cli", "vendor"}, file_pattern = ".lua" })
+local lua_sources = IO.retrieve_files("./kong", { exclude_dir_patterns = {"cli", "vendor", "filelog"}, file_pattern = ".lua" })
```
Comment (thefosk) [inline spec/unit/stub_coverage_spec.lua]: Temporary fix for busted segmentation fault bug.

**Comment by thefosk** [comment]
There is a problem with busted and LuaJIT (https://github.com/Olivine-Labs/busted/issues/426), so in order to make it work I have adopted two dirty fixes that I hope to remove soon:

* While the distribution package includes Luarocks + LuaJIT, becuase of that bug I had to bring back Lua when running busted (the `sed` line in `.travis.yml`). We can simply remove this line once the seg fault bug has been fixed.
* Because busted is forced to run with Lua (and not LuaJIT), I had to exclude `filelog` from the coverage. Once the seg fault in busted has been fixed, we can remove this exclusion along with the `sed` command in `.travis.yml`

**Comment by thibaultCha** [comment]
I have been running busted with LuaJIT for a few months now and never had any issue.

**Pull request merged**